eZ/Bundle/EzPublishCoreBundle/Imagine/VariationPathGenerator/WebpFormatVariationPathGenerator.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Imagine\VariationPathGenerator;
```

Comment:
Please move it to the new namespace. Otherwise you'd have to add current namespace to the compatibility layer.